### PR TITLE
adding espresso send command

### PR DESCRIPTION
**Pull request by DiegoSena**

### CODE REVIEW

**Comment by sandhilt** [comment]
Check this @ZzzzHui

File: internal/sequencers/espresso/espresso_client.go [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+		return "", fmt.Errorf("Error sending input to Espresso: %v", err)
+	}
+
+	fmt.Println("Input sent to Espresso successfully!")
+	return response, nil
+}
+
+func fetchNonce(client *ethclient.Client, privateKey *ecdsa.PrivateKey) (uint64, error) {
+	fromAddress := crypto.PubkeyToAddress(privateKey.PublicKey)
+	nonce, err := client.PendingNonceAt(context.Background(), fromAddress)
```
Comment (ZzzzHui) [inline internal/sequencers/espresso/espresso_client.go:146]: This looks strange honestly. How do you use `ethclient` to fetch nonce and use it as Espresso nonce? They are 2 different things in my opinion.
In the browser front end, we query graphql for the nonce like [this](https://github.com/ZzzzHui/frontend-web-cartesi/blob/24fa6d1b3f43f29ea716c27a5c2b7c04b0d2bb94/src/Input.tsx#L87). Feel free to let me know what you think :)

Comment (DiegoSena) [reply internal/sequencers/espresso/espresso_client.go]: Hey @ZzzzHui  you are right. Updated the code to match your example.

**Review by sandhilt — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**